Add Incident 2194 Entity Store Broken Transform to known issues (#3754)
elastic/docs-content: release-notes/elastic-security/known-issues.md

% :::

:::{dropdown} Entity store transform is unavailable 

> 

Reviewer: ```suggestion
:applies_to: stack: ga 9.2.0
```

Reviewer: I think this is the syntax you need for the dropdown applies_to badge: https://docs-v3-preview.elastic.dev/elastic/docs-builder/tree/main/syntax/dropdowns#with-applies_to-badge

You might need to remove the other `Applies to` reference in plain text

Reviewer 2: I'm not sure if we should introduce the dropdown applies_to in this case, since AFAIK, we'll be removing the option to specify patches in applies_to tags in the near future.

Applies to: 9.2.0

**Details**

> A new feature was introduced to entity store in 9.2.0 that caused the transform to scan for non-existent indices.

Reviewer: ```suggestion
A new feature introduced to the entity store in 9.2.0 caused the transform to scan for nonexistent indices.
```

Minor rewording for clarity.
